release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1484,6 +1484,12 @@ With this release, `oc` commands now default to obtaining the credentials from P
 
 Additionally, the `oc registry login` command now stores credentials in the Podman configuration locations instead of the Docker configuration file locations.
 
+[discrete]
+[id="ocp-4-14-ovnic-default-range"]
+=== Open Virtual Network Infrastructure Controller default range
+
+Previously, the IP address range `168.254.0.0/16` was the default IP address range that the Open Virtual Network Infrastructure Controller used for the transit switch subnet. With this update, the Controller uses `100.88.0.0/16` as the default IP address range. Do not use this IP range in your production infrastructure network. (link:https://issues.redhat.com/browse/OCPBUGS-20261[*OCPBUGS-20261*])
+
```

Comment:
The already-allocated IP range, `168.254.0.0/16`, was found before 4.14 went GA, so no released openshift version ever used it. Let's not refer to it, it was only used in the nightly versions before 4.14.0. There's no need for the customer to see the bug either, for the same reason.

Reply:
I'd add that it refers to the transit switch, so something like: ` The Controller uses 100.88.0.0/16 as the default IP address range for the newly-introduced transit switch subnet`

In 4.15 it's no longer "newly-introduced" though :)